### PR TITLE
feat: create basic layout

### PR DESCRIPTION
**Pull request by YanaK99**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-9ndCyUaIbzAi2FUVXJi0CjmCapSmO7SnpJef0486qhLnuZ2cdeRhO02iuK6FUUVM" crossorigin="anonymous">
```
Comment (denissosnowsky) [inline public/index.html]: Plz, replace bootstrap with Material UI
Can we somehow import styles from node_modules instead from index.html link tag

File: src/App.css [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
-  pointer-events: none;
+    height: 40vmin;
+    pointer-events: none;
 }
 
 @media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+    .App-logo {
```
Comment (denissosnowsky) [inline src/App.css]: we dont need it anymore, plz delete

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,40 @@
...
+    color: white;
+    font-size: calc(10px + 2vmin);
 }
 
 .App-link {
-  color: #61dafb;
+    color: #61dafb;
 }
 
 @keyframes App-logo-spin {
```
Comment (denissosnowsky) [inline src/App.css]: the same as above comment

File: src/components/Header/Header.css [hunk 4]
```diff
@@ -0,0 +1,17 @@
+.container {
```
Comment (denissosnowsky) [inline src/components/Header/Header.css]: plz remove container to App.js file

File: src/components/Navigation/Navigation.js [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+const Navigation = () => {
+    const [isOpen, setIsOpen] = useState(false);
+
+    const openNav = () => {
+        setIsOpen(true);
+    };
+
+    const closeNav = () => {
+        setIsOpen(false);
+    };
```
Comment (denissosnowsky) [inline src/components/Navigation/Navigation.js:13]: Plz press Enter

**Review by denissosnowsky — COMMENTED** [review]
Plz check comments

File: public/index.html (hunk 1 above)
Comment (YanaK99) [reply public/index.html]: fixed

File: src/App.css (hunk 2 above)
Comment (YanaK99) [reply src/App.css]: i did it

File: src/App.css (hunk 3 above)
Comment (YanaK99) [reply src/App.css]: ok

File: src/components/Header/Header.css (hunk 4 above)
Comment (YanaK99) [reply src/components/Header/Header.css]: +

File: src/components/Navigation/Navigation.js (hunk 5 above)
Comment (YanaK99) [reply src/components/Navigation/Navigation.js:13]: yeap

**Review by denissosnowsky — APPROVED** [review]

**Pull request merged**